tmcprototype/CentralNode/CentralNode/CentralNode.py

```diff
@@ -281,14 +312,14 @@ def delete_device(self):
     def read_telescopeHealthState(self):
         # PROTECTED REGION ID(CentralNode.telescope_healthstate_read) ENABLED START #
         """ Returns the Telescope health state."""
-        if ((self._subarray1_health_state == 1) | (self._subarray2_health_state == 1)):
-            self._telescope_health = 1
-        elif ((self._subarray1_health_state == 2) & (self._subarray2_health_state == 2)):
-            self._telescope_health = 2
-        elif ((self._subarray1_health_state == 0) & (self._subarray2_health_state == 0)):
-            self._telescope_health = 0
-        else:
-            self._telescope_health = 3
+        # if ((self._subarray1_health_state == 1) | (self._subarray2_health_state == 1)):
```

Comment:
Remove commented code or add TODO block if required in future.

Reply:
Done. The commit id is: https://github.com/ska-telescope/tmc-prototype/commit/dafa12fbac0e08db8ec3ec703afb99ba45db38c1